l10n_eu_product_adr/README.rst

```diff
@@ -46,6 +46,14 @@ delivery
 May more complex name generation needed for computing shipping name, base on
 dangerous class
 
+Changelog
+=========
+
+13.0.2.0.1 (2021-06-08)
```

Comment:
next version should be 2.0.0, here just say "from version 2.x.x" :man_shrugging:

Reply:
Done. The bot will do the rest, now.